requirements.txt

```diff
@@ -7,4 +7,7 @@ django-localflavor==3.1
 selenium==3.141.0
 webdriver-manager==3.8.5
 python-dateutil==2.8.2
-braintree==3.59.0
\ No newline at end of file
+pandas
```

Comment:
panda requirement line is duplicated, delete the one that does not indicate the version.

Reply:
Sorry for both errors, they are already solved